src/views/Sjofn/Homework6.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div>
+    <h1>Homework 6</h1>
+
+    <div>
+      <label for="ingredients">Ingredients: </label>
+      <input
+        id="ingredients"
+        v-model="ingredients"
+        type="text"
+        placeholder="Ingredients"
+        autofocus
+      >
+    </div>
+
+    <div>
+      <label for="dish">Dish: </label>
+      <input
+        id="dish"
+        v-model="dish"
+        type="text"
+        placeholder="Dish"
+      >
+    </div>
+
+    <button
+      :disabled="isLoading"
+      @click="fetchRecipe"
+    >
+      Search
+    </button>
+    <hr>
+    <table
+      v-if="results.length"
+      style="width:100%"
+    >
+      <tr>
+        <th>
+          Title
+        </th>
+        <th>
+          Link
+        </th>
+        <th>
+          Ingredients
+        </th>
+        <th>
+          Image
+        </th>
+      </tr>
+      <tr
+        v-for="result in results"
+        :key="result.href"
+      >
+        <td>
+          {{ result.title }}
+        </td>
+        <td>
+          {{ result.href }}
+        </td>
+        <td>
+          {{ result.ingredients }}
+        </td>
+        <td>
+          <img
+            v-if="result.thumbnail.length"
+            :src="result.thumbnail"
+          >
+        </td>
+      </tr>
+    </table>
+    <div
+      id="results"
+    >
+      <Loader v-if="isLoading" />
+      <pre v-else>{{ result }}</pre>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+import Loader from '../../components/Loader'
+export default {
+  name: `Homework6`,
+  components: { Loader },
+  data () {
+    return {
+      ingredients: '',
+      dish: '',
+      isLoading: false,
+      results: []
+    }
+  },
+  computed: {
+    isSearchButtonDisabled () {
```

Comment:
This computed property is never used. Why did you add it then?
Please, either remove it or use it somewhere in the component